lambda/export-variance/invokable/index.js

```diff
@@ -282,4 +282,50 @@ async function uploadToS3(csvData) {
   }
   logger.debug("Uploaded to S3");
 
+}
+
+function convertMonth(monthNumber){
+  let monthName;
+
+  switch (monthNumber) {
+    case 1:
+      monthName = 'January';
+      break;
+    case 2:
+      monthName = 'February';
+      break;
+    case 3:
+      monthName = 'March';
+      break;
+    case 4:
+      monthName = 'April';
+      break;
+    case 5:
+      monthName = 'May';
+      break;
+    case 6:
+      monthName = 'June';
+      break;
+    case 7:
+      monthName = 'July';
+      break;
+    case 8:
+      monthName = 'August';
+      break;
+    case 9:
+      monthName = 'September';
+      break;
+    case 10:
+      monthName = 'October';
+      break;
+    case 11:
+      monthName = 'November';
+      break;
+    case 12:
+      monthName = 'December';
+      break;
+    default:
+      monthName = `Invalid month: ${monthNumber}`;
+  }
+  return monthName
```

Comment:
Could you use an enum for this?

Reply:
```
function getMonthName(monthNumber) {
  const months = [
    'January', 'February', 'March', 'April', 'May', 'June',
    'July', 'August', 'September', 'October', 'November', 'December'
  ];

  if (monthNumber >= 1 && monthNumber <= 12) {
    return months[monthNumber - 1];
  } else {
    return 'Invalid month number';
  }
}

// Example usage:
const monthNumber = 5; // May
const monthName = getMonthName(monthNumber);
console.log(monthName); // Output: "May"
```

Reply:
Alternatively:

```
function getMonthName(monthNumber) {
  if (monthNumber >= 1 && monthNumber <= 12) {
    const date = new Date(2000, monthNumber - 1, 1); // Year 2000 is an arbitrary year.
    return new Intl.DateTimeFormat('en-US', { month: 'long' }).format(date);
  } else {
    return 'Invalid month number';
  }
}

// Example usage:
const monthNumber = 5; // May
const monthName = getMonthName(monthNumber);
console.log(monthName); // Output: "May"

```

Reply:
Updated using your first example. Tested locally, requesting re-review!